### ISSUE TITLE
multiple app_directory on same connection?

### ISSUE THREAD

**Issue by navihtot** [rocketeers/rocketeer#514]
Can you have multiple app_directoryies on same connection?
I have many cutomers using same app and my installs are all on same production server only in different folders.
I know you can have multiple stages which creates different subfolders i.e. my_app/one_stage, my_app/another_stage but can i have rocketeer place apps in different folders under /var/www/
i.e. /var/www/client_one, /var/www/client_another etc.

not sure if this is a place for questions but I didnt find another...

tnx

**Comment by Anahkiasen** [comment]
I don't think you can no. What you can do though is define a multiserver connection but which each entry actually pointing to the same server, but with different `app_directories`
